pyscf/adc/uadc.py

```diff
@@ -56,12 +56,12 @@ def kernel(adc, nroots=1, guess=None, eris=None, verbose=None):
 
     if adc.verbose >= logger.INFO:
         if nroots == 1:
-            logger.info(adc, '%s root %d    Energy (Eh) = %.8f    Energy (eV) = %.8f    Spec factors = %.8f',
-                         adc.method, 0, E, E*27.2114, spec_factors)
+            logger.info(adc, '%s root %d    Energy (Eh) = %.8f    Energy (eV) = %.8f    Spec factors = %.8f    conv = %s',
```

Comment:
If there are roots in UADC and RADC that are not converged (at least 1), we should print a warning message about that before printing the results